[FEATURE#11866] Ajout des test behat

gdubost opened the pull request:
[x] Ajout d'une "fake app" pour les tests
[x] Ajout des dependances requises dans composer.json (behat ...)
[x] Création du FeatureContext
[x] Écriture des tests

Commit:
[FEATURE#11866] Ajout des test behat

[x] Ajout d'une "fake app" pour les tests
[x] Ajout des dependances requises dans composer.json (behat ...)
[x] Création du FeatureContext
[x] Écriture des tests

SparSio commented on TestCheck/App.php:
```diff
@@ -0,0 +1,19 @@
+
+class App extends \Silex\Application
+{
+    use \Silex\Application\SecurityTrait;
```
pk ?

SparSio commented:
profite en pour faire ça https://scrutinizer-ci.com/g/etna-alternance/composer-check-service-provider/inspections/5bb4e00a-985e-45f3-99e5-347fe27ccfda/patches

Commit:
[FEATURE#11866] Ajout des test behat

    [x] Ajout d'une "fake app" pour les tests
    [x] Ajout des dependances requises dans composer.json (behat ...)
    [x] Création du FeatureContext
    [x] Écriture des tests
    [x] Ajout de drone

[FEATURE#11866] Ajout des test behat

   - [x] Ajout d'une "fake app" pour les tests
   - [x] Ajout des dependances requises dans composer.json (behat ...)
   - [x] Création du FeatureContext
   - [x] Écriture des tests
   - [x] Ajout de drone
   - [x] Ajout du coverage
